### ISSUE TITLE
Optimal Training Settings

### ISSUE THREAD

**Issue by eehoman** [sumantrapatnaik/WebNLG_NLP#6]
Experiment with network initialization settings. Dropout, epochs, batch size? Ties in with the network topology issue.

**Comment by eehoman** [comment]
Most network settings work best as-is, aside from the dropout settings. Putting it to 0.5 produces marginally better results. Expected, since a dropout of 0.5 is the recommenced initial setting.
